motion_predict/src/predict_ctrv.cpp

```diff
@@ -81,21 +86,23 @@ carma_perception_msgs::msg::PredictedState buildPredictionFromCTRVState(const CT
   pobj.predicted_position.position.z = original_pose.position.z;
 
   // Map orientation
-  Eigen::Quaternionf original_quat(original_pose.orientation.w, original_pose.orientation.x,
-                                   original_pose.orientation.y, original_pose.orientation.z);
-  Eigen::Vector3f original_rpy = original_quat.toRotationMatrix().eulerAngles(0, 1, 2);
-
-  auto vel_angle_and_mag = localVelOrientationAndMagnitude(original_twist.linear.x, original_twist.linear.y);
-
-  Eigen::Quaternionf final_quat;
-  final_quat = Eigen::AngleAxisf(original_rpy[0], Eigen::Vector3f::UnitX()) *
-               Eigen::AngleAxisf(original_rpy[1], Eigen::Vector3f::UnitY()) *
-               Eigen::AngleAxisf(state.yaw - std::get<0>(vel_angle_and_mag), Eigen::Vector3f::UnitZ());
-
-  pobj.predicted_position.orientation.x = final_quat.x();
-  pobj.predicted_position.orientation.y = final_quat.y();
-  pobj.predicted_position.orientation.z = final_quat.z();
-  pobj.predicted_position.orientation.w = final_quat.w();
+  pobj.predicted_position.orientation = original_pose.orientation;
+  //  Eigen::Quaternionf original_quat(original_pose.orientation.w, original_pose.orientation.x,
```

Comment:
Can we remove this commented out code block?

Reply:
removed